packages/js/scripts/size-limit.mjs

```diff
@@ -15,7 +15,7 @@ const modules = [
   {
     name: 'UMD minified',
     filePath: umdPath,
-    limitInBytes: 145_000,
+    limitInBytes: 150_000,
```

Comment:
why do we have to change this?

Reply:
This caused my build to fail and I thought it was an issue from a previous commit which got merged before ci. I will check again